### ISSUE TITLE
instance of pip installed outside the current conda environment is used.

### ISSUE THREAD

**Issue by ss62171** [MDAnalysis/UserGuide#63]
**Actual Behavior** 
According to [User guide](https://www.mdanalysis.org/UserGuide/contributing_code.html#building-mdanalysis) if we install MDA using `conda` environment and say I try to install some dependency using `pip` without installing it in my `conda` environment and it might be possible to have `pip` installed outside a `conda` environment which causes these module to install in global env.

**Expected Behavior**
using `pip` these packages must install in `conda` environment.

**Solution**
```python
# if using pip
pip install cython numpy mmtf-python mock six biopython \
 networkx cython matplotlib scipy griddataformats \
 hypothesis gsd codecov "seaborn>=0.7.0,<=0.9" \
 netcdf4 scikit-learn "joblib>=0.12" parmed psutil pytest
```
add a note at top of this like:
if using pip make sure you have `pip` installed in your `conda` environment

**Comment by RMeli** [comment]
My understanding of `# if using pip` is that it refers to `Creating a development environment - With pip and virtualenv`, so in that case `conda` would not be installed in the first place. In the same cell you have the same instructions for `conda` (why use `pip` if you are within a `conda` environment and if packages can be installed with `conda`?):

```bash
# if using conda
conda install -c biobuilds -c conda-forge \
 cython numpy mmtf-python mock six biopython \
 networkx cython matplotlib scipy griddataformats \
 hypothesis gsd codecov "seaborn>=0.7.0,<=0.9" \
 clustalw=2.1 netcdf4 scikit-learn "joblib>=0.12"\
 psutil pytest
# if using conda with python 3.7 or 3.8, also run
conda install -c conda-forge parmed
``` 

However, I do agree that `pip` is needed to install MDanalysis even when using `conda` (and some packages are only present on PyPI), therefore adding `pip` to the list of packages installed with `conda` might be useful.

**Comment by RMeli** [comment]
This should be fixed with #64 .
